filecoin-proofs/examples/ffi/main.rs

```diff
@@ -128,9 +141,9 @@ unsafe fn sector_builder_lifecycle(use_live_store: bool) -> Result<(), Box<Error
     let sizes = if use_live_store {
         ConfigurableSizes {
             sector_class: FFISectorClass {
-                sector_size: FFISectorSize_SSB_TwoHundredFiftySixMiB,
-                seal_proof_partitions: FFISealProofPartitions_SPP_Two,
-                post_proof_partitions: FFIPoStProofPartitions_PPP_One,
+                sector_size: 266338304,
```

Comment:
I don't understand how this number is calculated.

I believe this should correspond to the number of bytes in 256MiB, which is 268435456.

If that's as intended, please write this in a way which makes the derivation clear. For examples `1024 * 1024 * 256` or `1 << 28`.

It almost looks as though this might be the 'raw data size' (i.e. before bit-padding), but below you use 1024 for 1KiB. Without other information, I assume `sector_size` is the sector size, not the amount of original data a sector can hold but rather the power-of-two sector size.

Please clarify.

Reply:
> I believe this should correspond to the number of bytes in 256MiB, which is 268435456.

Yes, you are correct. The value 266338304 is computed from 268435456 minus bit padding. This is a bug, now fixed.

> If that's as intended, please write this in a way which makes the derivation clear. For examples 1024 * 1024 * 256 or 1 << 28.

Done.